docs/api-reference/data-management-api.md

```diff
@@ -46,6 +47,22 @@ When you use these APIs concurrently with an indexing task or a kill task, the b
 Druid terminates some segments and marks others as used.
 Furthermore, it is possible that all segments could be unused, yet an indexing task might still be able to read data from these segments and complete successfully.
 
+### New Overlord APIs
+
+All of the following APIs, except [Segment deletion](#segment-deletion) have been moved from the Coordinator to the Overlord as it is the service responsible for performing actions on segment metadata on behalf of indexing tasks.
+This makes the Overlord the single source of truth for segment metadata, thus ensuring a consistent view across the Druid cluster and allowing the Overlord to cache metadata to improve performance.
+The corresponding Coordinator APIs are now deprecated and will be removed in a future release. For the time being, they simply redirect the request to the Overlord.
+The same request payloads may be used for both versions of any API and the response payload has changed only in a couple of cases.
+
+|API|Method|Deprecated Coordinator path|New Overlord path|Change in response payload|
+|---|------|---------------------------|-----------------|--------------------------|
+|[Mark a single segment as used](#mark-a-single-segment-as-used)|`POST`|`/druid/coordinator/v1/datasources/segments/{segmentId}`|`/druid/coordinator/v1/datasources/segments/{segmentId}`|Yes (old: `{"segmentStateChanged": true}`, new: `{"numChangedSegments": 1}`)|
+|[Mark a single segment as unused](#mark-a-single-segment-unused)|`DELETE`|`/druid/coordinator/v1/datasources/{datasource}/segments/{segmentId}`|`/druid/indexer/v1/datasources/{datasource}/segments/{segmentId}`|Yes (old: `{"segmentStateChanged": true}`, new: `{"numChangedSegments": 1}`)|
```

Comment:
For backwards compatibility & to minimize API differences, could we also include `segmentStateChanged` in the response returned by the overlord-based single segment API (similar to the coordinator one)? The same would help with the above POST API as well

Reply:
Yeah, that's a good point. Let me see if I can fix it up.

Reply:
Actually, it looks like the `SegmentUpdateResponse` [returned](https://github.com/apache/druid/blob/master/server/src/main/java/org/apache/druid/rpc/indexing/SegmentUpdateResponse.java#L53) by the Overlord already includes `segmentStateChanged`. If so, then I think the documentation here is slightly confusing - a "Change in response payload" suggests that I would need to handle the responses differently, although I think that may not be necessary since only new fields are being added in the response.

Reply:
Oh, awesome! I forgot that I had made it backwards compatible 😛 
Let me fix up the release note and the docs too.